src/hb.hh

```diff
@@ -515,20 +515,6 @@ typedef uint64_t hb_vector_size_impl_t;
 #define VAR 1
```

Comment:
Not related to this, would be nice to get rid of this also sometime and replace all the uses with UnsizedArrayOf

Reply:
Most uses of this should be replaced with UnsizedArrayOf, yes.  But it will remain in implementation of UnsizedArrayOf, ArrayOf, etc.  There's very few instances.

Reply:
Or using UnsizedArrayOf inside ArrayOf and others and having only one of them, you said that once before IIRC